src/Repository/OrderStateRepository.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Mollie\Repository;
+
+use Db;
+
+class OrderStateRepository
+{
+    public function deleteMollieStatuses($name) {
```

Comment:
why you pass name when function is called `deleteMollieStatuses`  ? 

keep mollie name inside and not make it passable. Name function just `deleteStatuses`